src/citrine/resources/design_space.py

```diff
@@ -127,6 +128,30 @@ def restore(self, uid: Union[UUID, str]) -> DesignSpace:
         entity = self.session.put_resource(url, {}, version=self._api_version)
         return self.build(entity)
 
+    def _list_base(self, *, per_page: int = 100, archived: Optional[bool] = None):
+        filters = {}
+        if archived is not None:
```

Comment:
nit: This is use of `True`, `False`, and `None` is a bit unintuitive for me. In that it's unclear that the True and False are filters and that `None` is no filter. Not a huge deal, but maybe a quick enum for the three different archive_status' might be nice?

Reply:
`None` always signifies the lack of specification: https://docs.python.org/3/c-api/none.html#c.Py_None

The right way to think of this is that omitting `archived` is no filter, which we indicate in Python with `None`.